### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -S po -L
gir,lene,teh,taht,youre,jsut,thier,tihs,filterin,readded,ciph`

### CODE REVIEW

File: pynicotine/gtkgui/frame.py [hunk 1]
```diff
@@ -1463,7 +1463,7 @@ def create_dialog():
                 callback=self.shares_unavailable_response
             ).show()
 
-        # Avoid dialog appearing deactive if invoked during rescan on startup
+        # Avoid dialog appearing deactivate if invoked during rescan on startup
```
Comment (mathiascode) [inline pynicotine/gtkgui/frame.py:1466]:
```suggestion
        # Avoid dialog appearing inactive if invoked during rescan on startup
```

File: NEWS.md [hunk 2]
```diff
@@ -300,10 +300,10 @@ WINDOWS USERS: The installer format has changed in Nicotine+ 3.2.0 and above. If
  * Text Entry should validate and execute upon input when focus moves away (Result Filters) ([#1498](https://github.com/nicotine-plus/nicotine-plus/issues/1498))
  * Text Entry should respond to a zero-length string created by any keypress event to force clear the filter (Result Filters) ([#1499](https://github.com/nicotine-plus/nicotine-plus/issues/1499))
  * Fix missing Alt+R accelerator for Result Filter bar show/hide button in Search Files ([#1500](https://github.com/nicotine-plus/nicotine-plus/issues/1500))
- * Redundent Find pop-up TextBox in Search Files TreeView widget hinders Ctrl+F so it needs to be disabled ([#1501](https://github.com/nicotine-plus/nicotine-plus/issues/1501))
+ * Redundant Find pop-up TextBox in Search Files TreeView widget hinders Ctrl+F so it needs to be disabled ([#1501](https://github.com/nicotine-plus/nicotine-plus/issues/1501))
```
Comment (slook) [inline NEWS.md:303]: I'm not sure if I support the fixing of typos that are direct hypertext references to GitHub Issue titles.

Comment (mathiascode) [reply NEWS.md:303]: I can update the issue titles on GitHub.

**Review by mathiascode — APPROVED** [review]
Thanks!

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**